# reason: configure looks for the wrong filename
# note: should be fixed in the next release

--- configure
+++ configure
@@ -34701,7 +34701,7 @@
     ;;
     5)
       QT_MODULES="Qt5Core Qt5Gui Qt5Network Qt5OpenGL Qt5PrintSupport"
-      octave_qscintilla_libnames="qscintilla2-qt5 qt5scintilla2"
+      octave_qscintilla_libnames="qscintilla2-qt5 qt5scintilla2 qscintilla2_qt5"
     ;;
     *)
       as_fn_error $? "Unrecognized Qt version $qt_version" "$LINENO" 5
